mods/3d_armor/wieldview/depends.txt

```diff
@@ -1,2 +1 @@
-default
 3d_armor
```

Comment:
This file is overridden by `mods/3d_armor/wieldview/mod.conf`, isn't it? Should we remove it?

Reply:
I can remove it as well, but I'm not sure.
I agree to remove it.

Reply:
You can remove it, I'm sure.

Reply:
I'll remove all `depends.txt` and `description.txt` on `3d_armor`.